swarm/pss/types.go

```diff
@@ -91,29 +93,65 @@ type pssDigest [digestLength]byte
 
 // conceals bitwise operations on the control flags byte
 type msgParams struct {
-	raw bool
-	sym bool
+	raw bool // unenctypted mode
+	sym bool // symmetric encryption
+	nhr int  // neighbourhood addressing mode based on minumim proximity order
```

Comment:
Why `int`? Doesn't `byte` suffice?

Reply:
i don't understand. Can you have proximity orders > 256?

Reply:
ok it could be uint8 but then I need to convert elsewhere